packages/web/src/shared/AwesomeFade.tsx

```diff
@@ -0,0 +1,72 @@
+import { keyframes } from '@emotion/core'
+import * as React from 'react'
+import Reveal from 'react-awesome-reveal'
+
+type Direction = 'up' | 'right'
```

Comment:
'X' or 'Y' might make more sense here since we're using translations

Reply:
yeah originally i was match the API of the react-awesome-reveal but this makes sense